src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpResponseStream.Managed.cs

```diff
@@ -162,7 +162,7 @@ internal void InternalWrite(byte[] buffer, int offset, int count)
 
         private async Task InternalWriteIgnoreErrorsAsync(byte[] buffer, int offset, int count)
         {
-            try { await _stream.WriteAsync(buffer.AsMemory(offset, count)).ConfigureAwait(false); }
+            try { await _stream.WriteAsync(buffer.AsMemory(offset, count), default).ConfigureAwait(false); }
```

Comment:
Why was this change needed?  There's no token in context here.

Reply:
Removed. My bad. Not sure why I added it here. It was showing up before the roslyn-analyzers fix, but it's not showing up anymore.